model_compression_toolkit/core/keras/quantizer/fake_quant_builder.py

```diff
@@ -69,9 +69,9 @@ def power_of_two_quantization(activation_n_bits: int,
     activation_is_signed = quantization_params.get(SIGNED)
 
     if activation_threshold is None or activation_is_signed is None:
-        return None
+        return None  # pragma: no cover
```

Comment:
for all "return None": if it's an error, why not raise an excwption. if not, maybe we should test it